main.py

```diff
@@ -82,6 +86,14 @@ def get_parameters():
         _args.partitioner_class = os.environ["PARTITIONER_CLASS"]
     if "TOPICS_DIR" in os.environ:
         _args.topics_dir = os.environ["TOPICS_DIR"]
+    if "ERRORS_TOLERANCE" in os.environ:
+        _args.errors_tolerance = os.environ["ERRORS_TOLERANCE"]
+    if "ERRORS_DEADLETTERQUEUE_TOPIC_NAME" in os.environ:
+        _args.errors_deadletterqueue_topic_name = os.environ["ERRORS_DEADLETTERQUEUE_TOPIC_NAME"]
+    if "ERRORS_DEADLETTERQUEUE_TOPIC_REPLICATION_FACTOR" in os.environ:
+    _args.errors_deadletterqueue_topic_replication_factor = int(_args.errors_deadletterqueue_topic_replication_factor) = os.environ["ERRORS_DEADLETTERQUEUE_TOPIC_REPLICATION_FACTOR"]
```

Comment:
indentation is wrong and you can't convert to int at this point. It should follwo the pattern of `_args.initial_wait_time`